docs/make-ckan-live.md

```diff
@@ -0,0 +1,46 @@
+# Making CKAN Live
+
+## Point your live URL to the correct DNS
+
+- Go to the AWS EC2 Dashboard.
+- Select your EC2 instance and copy the Public DNS URL.
+  ![image-20200720113944879](../assets/public_dns.png)
+- On your hosting service, create a `CNAME` record with your live site URL. Point it to the Public DNS URL you just copied.
+  - Example: `CNAME		data.openpeoria.com`
+
+
+
+## Point CKAN Config to your live URL
+
+- SSH into your CKAN EC2 instance.
+- Open the `production.ini` file and change the `ckan.site_url` variable.
+  ```bash
+  sudo vim /lib/ckan/default/src/ckan/production.ini
+  ```
+  ```ini
+  ckan.site_url = {your_live_site}
+  # example - ckan.site_url = https://data.openpeoria.com
+  ```
+
+
+
+## Set up HTTPS
+
+### Add an Acme Challenge
+
+To get the LetsEncrypt SSL certificate, you will need to create a `TXT` record with an Acme challenge.
+
+- Add a `TXT` record to your hosting service with a key of `_acme-challenge.{your_live_site}`and a value of `Op6S2FCJ720AGVJGnYg4Hs00mCiUdm-bxg9Tcsgh1zE`.
+  - *Example*: ![img](../assets/txt_record.png)
+
+### Get New Certificate
+
+You will need a new SSL certificate for your live domain name. You can set that up by following [these instructions](https://handbook.nerevu.com/ckan-https.html).
+
+- **IMPORTANT**: When you get to [this section](https://handbook.nerevu.com/ckan-https.html#set-up-certbot-on-your-ec2-instance) that has you put a `cron` job in your crontab to update your SSL certificate automatically, be sure to delete the other cron job that is updating the old SSL certificate.
```

Comment:
This section should probably be moved up since the acme challenge is set (or maybe not) here.

Reply:
I think I'm going to move the Acme Challenge part to the HTTPS instructions I wrote and remove it from this document.